reference/features/known_limitations.md

```diff
@@ -1,36 +1,37 @@
+**Limitation:** The maximum number of terms taken into account for each [search query](/reference/features/search_parameters.md#query-q) is 10. **If a search query includes more than 10 words, all words after the 10th will be ignored.**
 
-This is because large queries can lead to long response times, and MeiliSearch's goal is always to provide a [fast search-as-you-type experience](/learn/what_is_meilisearch/philosophy.md#front-facing-search).
+**Explanation:** The more terms in a query, the longer it will take MeiliSearch to return matching documents. This goes against our goal of providing a [fast search-as-you-type experience](/learn/what_is_meilisearch/philosophy.md#front-facing-search).
 
 ### Database size
 
-MeiliSearch uses two databases: the first one for storage and the second one for updates.
-On launch LMDB needs to know the size that it can allocate on disk. This space will be reserved on disk for LMDB, thus MeiliSearch. This space will also be allocated as virtual memory.
+**Limitation:** The default maximum database size is __100GiB__. This size can be modified using the options `--max-mdb-size` & `--max-udb-size` as described in the [configuration reference](/reference/features/configuration.md#max-mdb-size).
 
-The maximum database size is by default __100GiB__ for each database. This size can be modified using the options `--max-mdb-size` & `--max-udb-size` as described in the [configuration guide](/reference/features/configuration.md#max-mdb-size).
+**Explanation:** MeiliSearch uses two databases: one for storage and one for updates. On launch, LMDB needs to know the maximum size that it will need to reserve on disk for both of them.
 
 ### Number of indexes
 
-You can create __up to 200 indexes__ in MeiliSearch. This limit has been hard set for performance reasons.
+**Limitation:** You can create __up to 200 indexes__ in MeiliSearch.
+
+**Explanation:** This limit has been hard-coded for performance reasons.
 
 ### Maximum words per attribute
 
-A maximum of __1000 words per attribute__ can be indexed. This means that if an attribute contains more than 1000 words, only the first 1000 words will be indexed and the rest will be silently ignored.
+**Limitation:** MeiliSearch can index a maximum of __1000 words per attribute__. If an attribute contains more than 1000 words, only the first 1000 words will be indexed and the rest will be silently ignored.
 
-This limit is enforced for relevancy reasons. The more words there are in a given attribute, the less relevant the search queries will be.
+**Explanation:** This limit is enforced for relevancy reasons. The more words there are in a given attribute, the less relevant the search queries will be.
 
 ## Other limitations
 
 ### Payload size
 
-The default limit for the payload size is around __100MB__. [This limit can be modified](/reference/features/configuration.md#payload-limit-size).
+**Limitation:** The default limit for the payload size is around __100MB__.
```

Comment:
@dichotommy how do you feel about `around 100MB`? I'm a bit bothered because it feels so imprecise. Although functionally similar, I'm tempted to replace it with `~100MB`.